singlestoredb/fusion/handlers/job.py

```diff
@@ -0,0 +1,594 @@
+          EXTENDED;
+    """
+
+    def run(self, params: Dict[str, Any]) -> Optional[FusionSQLResult]:
+        res = FusionSQLResult()
+        res.add_field('ExecutionID', result.STRING)
+        res.add_field('ExecutionNumber', result.INTEGER)
+        res.add_field('JobID', result.STRING)
+        res.add_field('Status', result.STRING)
+        res.add_field('ScheduledStartTime', result.DATETIME)
+        res.add_field('StartedAt', result.DATETIME)
+        res.add_field('FinishedAt', result.DATETIME)
+
+        jobs_manager = get_workspace_manager().organizations.current.jobs
+
+        executionsData = jobs_manager.get_executions(
+            params['job_id'],
+            params['from_start'],
+            params['to_end'],
+        )
+
+        if params['extended']:
+            res.add_field('SnapshotNotebookPath', result.STRING)
+
+            def fields(execution: Any) -> Any:
+                return (
+                    execution.execution_id,
+                    execution.execution_number,
+                    execution.job_id,
+                    execution.status.value,
+                    dt_isoformat(execution.scheduled_start_time),
+                    dt_isoformat(execution.started_at),
+                    dt_isoformat(execution.finished_at),
+                    execution.snapshot_notebook_path,
+                )
+        else:
+            def fields(execution: Any) -> Any:
+                return (
+                    execution.execution_id,
+                    execution.execution_number,
+                    execution.job_id,
+                    execution.status.value,
+                    dt_isoformat(execution.scheduled_start_time),
+                    dt_isoformat(execution.started_at),
+                    dt_isoformat(execution.finished_at),
+                )
+
+        res.set_rows([fields(execution) for execution in executionsData.executions])
+
+        return res
+
+
+ShowJobExecutionsHandler.register(overwrite=True)
+
+
+class ShowJobRuntimesHandler(SQLHandler):
+    """
+    SHOW JOB RUNTIMES;
+
+    Description
+    -----------
+    Shows the available runtimes for jobs.
+
+    Example
+    -------
+    The following command shows the available runtimes for jobs::
+
+        SHOW JOB RUNTIMES;
+    """
+
+    def run(self, params: Dict[str, Any]) -> Optional[FusionSQLResult]:
+        res = FusionSQLResult()
+        res.add_field('Name', result.STRING)
+        res.add_field('Description', result.STRING)
+
+        jobs_manager = get_workspace_manager().organizations.current.jobs
+
+        runtimes = jobs_manager.runtimes()
+
+        def fields(runtime: Any) -> Any:
+            return (
+                runtime.name,
+                runtime.description,
+            )
+
+        res.set_rows([fields(runtime) for runtime in runtimes])
+
+        return res
+
+
+ShowJobRuntimesHandler.register(overwrite=True)
+
+
+class DropJobHandler(SQLHandler):
+    """
+    DROP JOB job_id;
```

Comment:
Should we support multiple job IDs here?

Reply:
For this one it makes sense, I will implement it.